packages/manager/apps/pci-rancher/src/components/layout-helpers/Dashboard/Dashboard.tsx

```diff
@@ -76,6 +79,27 @@ const Dashboard: React.FC<DashboardLayoutProps> = ({ tabs, rancher }) => {
     EditAction.UpdateSoftware,
   );
 
+  const { currentTasks } = rancher;
+
+  useEffect(() => {
+    if (updateSoftwareResponseType === 'pending') {
+      setIsPendingUpdate(true);
+    }
+  }, [updateSoftwareResponseType]);
+
+  useEffect(() => {
+    if (currentTasks.length) {
+      setHasTaskPending(true);
+    }
+  }, [currentTasks]);
+
+  useEffect(() => {
+    if (hasTaskPending && currentTasks.length === 0) {
+      setIsPendingUpdate(false);
+      setHasTaskPending(false);
+    }
+  }, [currentTasks]);
```

Comment:
**question:** Why is this change included in `BaseLayout` changes?

Reply:
Because I create an attribute DashboardMessages attributes to pass it below the tabs section

Reply:
I will update rancher in an other PR to merge the component more easily